golem_messages/message/base.py

```diff
@@ -249,37 +249,38 @@ def serialize_header(self):
             self.encrypted,
         )
 
-    def serialize_slot(self, key, value):  # noqa pylint: disable=unused-argument, no-self-use
+    def serialize_slot(self, key, value):
         if isinstance(value, enum.Enum):
             return value.value
         if key in self.MSG_SLOTS:

```

Comment:
General observation: I find the class name `MessageSlot` (and consequently `MSG_SLOTS`) confusing. Perhaps this could be a good moment to update it to something unambiguous and describing its role more accurately (e.g. `MessageSlotDefinition` or `MessageTypeSlot`).